pkg/traces/remotewriteexporter/exporter_test.go

```diff
@@ -85,6 +85,22 @@ type mockManager struct {
 	instance *mockInstance
 }
 
+// ApplyConfigs is used to batch configurations for performance instead of the singular ApplyConfig
+func (m *mockManager) ApplyConfigs(configs []instance.Config) error {
+	failed := make([]instance.BatchFailure, 0)
+	for _, v := range configs {
+		err := m.ApplyConfig(v)
+		if err != nil {
+			failed = append(failed, instance.BatchFailure{
+				Err:    err,
+				Config: v,
+			})
+		}
+	}
+	return instance.CreateBatchApplyErrorOrNil(failed, nil)
```

Comment:
nit: this should call out to a mocked function instead of actually implementing something.